src/components/navigation/perPage/NavigationPerPageReducers.ts

```diff
@@ -0,0 +1,50 @@
+import { IReduxAction, IReduxActionPayload } from '../../../utils/ReduxUtils';
+import { PerPageActions } from './NavigationPerPageActions';
+import * as _ from 'underscore';
+
+export interface IPerPageState {
+  id: string;
+  perPage: number;
+}
+
+export const perPageInitialState: IPerPageState = {
+  id: undefined,
+  perPage: 10
+};
+
+export const perPageCompositeInitialState: IPerPageState[] = [];
+
+export const perPageReducer = (state: IPerPageState = perPageInitialState, action: IReduxAction<IReduxActionPayload>): IPerPageState => {
```

Comment:
It is not simply for the UTs. It is being used by reducers and mapDispatchToProps as well.